src/BlockTypesController.php

```diff
@@ -206,6 +206,7 @@ protected function get_block_types() {
 			'RatingFilter',
 			'ReviewsByCategory',
 			'ReviewsByProduct',
+			'SingleProductDetails',
```

Comment:
I recommend disabling this block in Post and Page editor by not registering them, as we did for all other blocks [recently introduced](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/src/BlockTypesController.php#L253-L257).

Reply:
This is smart! Thank you for sharing, I just fixed it